Formula/http-server.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class HttpServer < Formula
+  desc "A simple zero-configuration command-line HTTP server"
+  homepage "https://github.com/indexzero/http-server"
+  url "https://registry.npmjs.org/http-server/-/http-server-0.10.0.tgz"
+  sha256 "cf7bde2def672698f78463ce8bec54dfe6392c24552f2b1c5e2e5ce94428de23"
+  head "https://github.com/indexzero/http-server.git"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_match "usage: http-server [path] [options]", shell_output("#{bin}/http-server --help | head -n 1")
```

Comment:
This needs to be a test that verifies the software's functionality once installed.